### DISCUSSION TITLE
Set custom dashboard widget as a predefined

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15751] [Feature improvement]
Here (probably not only) I have some 15 roles which should have the same set of 30 widgets defined on the basis of filter.
As each widget needs to be defined separately, it brings A LOT of clicking to make it for each role.
What would be the suggestion to make life easier? Maybe option to convert custom widget to predefined?

**Comment by antcorro** [comment]
Hello I have the same problem.
How to share a dashboard widget with more roles?

Thanks

**Reply by vovpff** [reply]
This functionality unavailable now. Maybe in future this  will be added
